Tests/UnitTests/Networking/Backend/__Snapshots__/BackendPostAttributionDataTests/iOS12-testPostAttributesPutsDataInDataKey.1.json

```diff
@@ -1,16 +0,0 @@
-{
```

Comment:
I suspect this file is still needed to make iOS 12 tests pass.

Reply:
ah yes, my bad. updating